utopia_tree.py

```diff
@@ -0,0 +1,38 @@
+#!/bin/python3
+
+import math
```

Comment:
Do you need math?

Reply:
Was already imported in the code taken from Hackerrank.

Reply:
remove these?